internal/pkg/service/stream/storage/level/local/diskreader/config.go

```diff
@@ -8,16 +8,13 @@ type Config struct {
 	// WaitForVolumeIDTimeout defines how long to wait for the existence of a file with the ID,
 	// see Open function and Volume.waitForVolumeID method.
 	WaitForVolumeIDTimeout time.Duration
-	// FileOpener provides file opening.
+	// OverwriteFileOpener overwrites file opening.
 	// A custom implementation can be useful for tests.
-	FileOpener FileOpener
+	OverwriteFileOpener FileOpener
```

Comment:
For values ​​that cannot be easily serialized, we use `nil`, these are some additional settings for tests. I moved the creation of the default implementation to other places in the code.

Reply:
So I can use `encoding.Config` in the File/Slice entity.